comms/src/message/envelope.rs

```diff
@@ -130,20 +121,22 @@ impl MessageEnvelope {
     }
 
     /// Returns the Message deserialized from the body frame
-    pub fn message_body(&self) -> Result<Message, MessageError> {
+    pub fn to_message(&self) -> Result<Message, MessageError> {
         Message::from_binary(self.body_frame()).map_err(Into::into)
     }
 
     /// Returns the decrypted and deserialized Message from the body frame
-    pub fn decrypted_message_body<PK>(
+    pub fn to_decrypted_message<PK>(
```

Comment:
Nit: We have you renamed this? To_XX makes me think you are converting the body, which you are not doing.

Reply:
That's exactly what i'm doing ;) (decrypting and deserializing the body)